astrodb_utils/utils.py

```diff
@@ -6,19 +6,19 @@
 from pathlib import Path
 
 import requests
-import sqlalchemy.exc
 from astrodbkit.astrodb import Database, create_database
-from sqlalchemy import and_
 
 __all__ = [
     "AstroDBError",
     "load_astrodb",
     "internet_connection",
-    "ingest_instrument",
     "exit_function",
+    "get_db_regime",
 ]
 
 logger = logging.getLogger(__name__)
+msg = f"logger.parent.name: {logger.parent.name}, logger.parent.level: {logger.parent.level}"
+logger.info(msg)
```

Comment:
Does this mean whenever you import astrodb_utils.utils you get an INFO message? Since it's not part of a function I just wonder if it would happen and if it would be desired.

Reply:
changed to DEBUG.